tools/packaging/qemu/patches/5.1.x/0014-gitmodules-use-GitLab-repos-instead-of-qemu.org.patch

```diff
@@ -103,16 +104,7 @@ index 2bdeeacef8..08b1b48a09 100644
 +	url = 	https://gitlab.com/qemu-project/opensbi.git
  [submodule "roms/qboot"]
  	path = roms/qboot
--	url = https://git.qemu.org/git/qboot.git
-+	url = https://gitlab.com/qemu-project/qboot.git
- [submodule "meson"]
- 	path = meson
--	url = https://git.qemu.org/git/meson.git
-+	url = https://gitlab.com/qemu-project/meson.git
- [submodule "roms/vbootrom"]
- 	path = roms/vbootrom
--	url = https://git.qemu.org/git/vbootrom.git
-+	url = https://gitlab.com/qemu-project/vbootrom.git
+ 	url = https://github.com/bonzini/qboot
```

Comment:
https://gitlab.com/qemu-project/qboot instead of Paolo's own repo?

Reply:
> https://gitlab.com/qemu-project/qboot instead of Paolo's own repo?

AFAICT the repo on gitlab is a mirror of Paolo's. What's the motivation to switch to gitlab ?

Reply:
Not diverge from what's already upstream, see: https://github.com/qemu/qemu/blob/2c3e83f92d93fbab071b8a96b8ab769b01902475/.gitmodules#L58-L60:
```
[submodule "roms/qboot"]
	path = roms/qboot
	url = https://gitlab.com/qemu-project/qboot.git
```

Reply:
Well this will still diverge a bit because we won't add a meson submodule, but your point is fair so let's do the switch to keep the divergence minimalistic.

Reply:
> let's do the switch to keep the divergence minimalistic.

That's funny because I meant to keep the divergence from 5.1.0 minimalistic and since it's not on `git.qemu.org`, I figured it wasn't required. Anyway, I will update to the GitLab repo :)

Reply:
That was my take also but @fidencio is right : we want to be as close as possible from what's in main to ease subsequent backports.